src/main/java/com/sun/xml/messaging/saaj/soap/MessageImpl.java

```diff
@@ -33,6 +33,8 @@
 import com.sun.xml.messaging.saaj.util.*;
 import org.jvnet.mimepull.MIMEPart;
 
+import static java.lang.String.format;
```

Comment:
avoid using static imports, at least in this case I see no benefits in their usage, it just makes the code less readable

Reply:
Agree 👍 Fixed!